### PR TITLE
21638-Implementing-FT2Plugin-with-FFI-calls-in-the-image-side

### PR DESCRIPTION
**Pull request by tesonep**
I have made an experiment to implement the bogus FT2Plugin in the image side using FFI calls. 
The image is working fine, It should be tested more. 
I have tested it in 32 and 64 bits in Linux and OSX. 
It should be tested in Windows. 
And further in all the platforms.

### CODE REVIEW

File: src/Athens-Cairo/CairoFontMetricsProvider.class.st [hunk 1]
```diff
@@ -71,6 +71,7 @@ CairoFontMetricsProvider >> extentsOf: aString [
 { #category : #accessing }
 CairoFontMetricsProvider >> font: aFont [
 	font := aFont asFreetypeFont.
+	self halt.
```
Comment (dionisiydk) [inline src/Athens-Cairo/CairoFontMetricsProvider.class.st]: halt in code

**Comment by tesonep** [comment]
I will remove it. Clearly is never running that code... :)

File: src/Athens-Cairo/CairoFontMetricsProvider.class.st (hunk 1 above)
Comment (tesonep) [reply src/Athens-Cairo/CairoFontMetricsProvider.class.st]: I removed it. Thanks a lot.

**Comment by estebanlm** [comment]
crossing fingers :)

**Pull request merged**